Dockerfile

```diff
@@ -12,21 +12,15 @@ RUN apt update \
   ninja-build \
   python3 \
   vim \
+  curl \
  && apt autoremove
 
 WORKDIR /root
-RUN git clone --depth=1 --branch llvmorg-10.0.0 https://github.com/llvm/llvm-project.git
-RUN mkdir build \
- && cd build \
- && cmake -G Ninja -DCMAKE_BUILD_TYPE=Release \
-  -DLLVM_ENABLE_PROJECTS="clang;lld" \
-  -DLLVM_TARGETS_TO_BUILD="AArch64;ARM" \
-  /root/llvm-project/llvm \
- && ninja \
- && DESTDIR=/tmp/clang-install ninja install
+RUN curl -s -L https://fileserver.linaro.org/s/da26ygYptknxwSk/download -o clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04
```

Comment:
What is this URL? Any guarantee on how long it will remain available? How to build the file once it's not there, and more importantly, when Clang is updated upstream?

In fact I'd much rather depend on the links you provide in the commit description instead. If the steps are not trivial you may create a script and use it in the Dockerfile, something like that:
```
ADD get_clang.sh /root/get_clang.sh
RUN get_clang.sh /tmp/clang-install
```
```
#!/bin/sh
#
# get_clang.sh
# Download a release of Clang and extract it to $1

# Base compiler
CLANG_X86_64 = https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang+llvm-10.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz
# For the Aarch64 compiler-rt libraries
CLANG_AARCH64 = https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang+llvm-10.0.0-aarch64-linux-gnu.tar.xz
# For the ARMv7/Aarch32 compiler-rt libraries
CLANG_ARMV7A = https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/clang+llvm-10.0.0-armv7a-linux-gnueabihf.tar.xz

curl ...
tar ...
```